.github/workflows/release_linux_aarch64.yml

```diff
@@ -90,14 +90,14 @@ jobs:
         python -m pip install -q twine
         twine upload --verbose dist/*.whl --repository-url https://upload.pypi.org/legacy/ -u ${{ secrets.ONNXWEEKLY_USERNAME }} -p ${{ secrets.ONNXWEEKLY_TOKEN }}
 
-    - name: Verify ONNX with the latest numpy and protobuf
+    - name: Verify ONNX with the latest numpy (below version 2) and protobuf
```

Comment:
onnx must be built with numpy>=2 and test should run with both versions numpy<2 and numpy>=2.

Reply:
It's for the 1.16.2 release. The changes from https://github.com/onnx/onnx/pull/6196 would not be included in 1.16.2 according to current planning status.

Reply:
The upcoming ONNX 1.17 added support in June for numpy 2 via https://github.com/onnx/onnx/pull/6196. So it's not in the 1.16 ONNX series which split from main around March. 

The numpy 2 support PR is a large PR by itself and would require other additional PRs to cherry-pick cleanly into the 1.16 series. Due to the size and complexity we're treating numpy 2 as more a new feature (ie part of a 1.X update) and beyond scope for security/patch update (1.16.X)